include/cling/Utils/AST.h

```diff
@@ -50,6 +50,18 @@ namespace utils {
     ///
     bool IsWrapper(const clang::FunctionDecl* ND);
 
+    ///\brief Get the mangled name of a NamedDecl.
+    ///
+    ///\param [in]  ND - Try to mangle this decl's name.
+    ///\param [in]  GD - Provide the GlobalDecl if what is to be mangled is a
+    /// constructor or destructor.
+    ///
+    ///\returns The mangled or pure name (when ND should not be mangled) on
+    /// success or an empty string on failure.
+    ///
+    std::string maybeMangleDeclName(const clang::NamedDecl* ND,
+                                    const clang::GlobalDecl* GD = nullptr);
```

Comment:
Huh? Now I'm confused.
d14e9b4 is right below this comment section.
If it appears in any master it would be an interesting SHA1 collision.